backend/tracim_backend/applications/upload_permissions/lib.py

```diff
@@ -293,6 +295,7 @@ def upload_files(
                 workspace_in_context=workspace_lib.get_workspace_with_context(
                     upload_permission.workspace
                 ),
+                uploader_message=message,
```

Comment:
There is a problem about `message` variable:

In http views, this is a non required parameter:

``` python
class UploadDataFormSchema(marshmallow.Schema):
    message = StrippedString(required=False, allow_none=False)
    # ...
```

But as start of `upload_files` method, this is a required parameter:

``` python
    def upload_files(
        self,
        upload_permission: UploadPermission,
        uploader_username: str,
        message: str,
        # ...
```

So, as start of `upload_files` method, `message` variable is not tested as an Optional type. 

Maybe you have just ti change typing. If not, create a technical debt issue is to long to modify before 2.4.0.

Reply:
you 're right message typing is uncorrect.
It's just fixed.

Reply:
perfect